clients/python/girder_client/cli.py

```diff
@@ -177,21 +197,29 @@ def main(ctx, username, password, api_key, api_url, scheme, host, port, api_root
     input his/her password.
     """
     # --api-url and URL by part arguments are mutually exclusive
+    if not config:
+        paths = _loadConfigPaths()
+        if paths:
+            config = paths[0]
```

Comment:
I'm not sure whether the behavior here should be to only look at one config file, or just list them all in order of precedence and have ConfigParser attempt to read all of them. I'm leaning toward the latter, which would eliminate a lot of this custom logic.